site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -77,11 +77,10 @@ Notice that the `w` function takes two parameters - a widget class and an object
 ```ts
 export interface WidgetProperties {
 	key?: string;
-	bind?: any;
 }
 ```
 
-Both of these properties are optional, so we can pass an empty object for now. Next, we will replace the `Banner` class with the `App` as the root of our application.
+`key` is optional, so we can pass an empty object for now. Next, we will replace the `Banner` class with the `App` as the root of our application.
 
 {% aside 'Mandatory object for properties' %}
```

Comment:
Let's move this aside just above the previous paragraph as the current rendering is truncating the bottom of the aside. 
![screenshot 2018-03-04 07 10 58](https://user-images.githubusercontent.com/97291/36946522-392b3ac8-1f7b-11e8-825d-5820cbf2d598.png)